tests/integration/models/application.spec.coffee

```diff
@@ -191,36 +191,30 @@ describe 'Application Model', ->
 				promise = resin.models.application.getApiKey(999999)
 				m.chai.expect(promise).to.be.rejectedWith('Application not found: 999999')
 
+	describe 'when toggling device URLs', ->
+		beforeEach ->
+			resin.models.application.create('DeviceUrlsTestApp', 'raspberry-pi').then (application) =>
+				@application2 = application
+				resin.models.device.register(@application2.id, resin.models.device.generateUniqueKey())
+			.then (deviceInfo) =>
+				@deviceInfo2 = deviceInfo
```

Comment:
Is there a reason this needs the `2`? I can't see another `deviceInfo` in this file.